package.json

```diff
@@ -31,6 +31,7 @@
     "@types/rsvp": "^4.0.2",
     "babel-eslint": "^8.0.0",
     "broccoli-asset-rev": "^3.0.0",
+    "ember-a11y-testing": "^3.0.2",
```

Comment:
@mike-north, alternatively we can pin ember-a11y-testing to v2.0.0, which supported node v8. As far as I can tell, there are no impacts to the a11y results.